packages/material-ui-lab/src/MonthPicker/PickersMonth.tsx

```diff
@@ -1,63 +1,93 @@
 import * as React from 'react';
 import clsx from 'clsx';
-import Typography from '@material-ui/core/Typography';
-import { MuiStyles, StyleRules, WithStyles, withStyles } from '@material-ui/core/styles';
+import Typography, { TypographyTypeMap } from '@material-ui/core/Typography';
+import {
+  experimentalStyled,
+  unstable_useThemeProps as useThemeProps,
+} from '@material-ui/core/styles';
+import { unstable_composeClasses as composeClasses } from '@material-ui/unstyled';
+import { OverridableComponent } from '@material-ui/core/OverridableComponent';
 import { onSpaceOrEnter } from '../internal/pickers/utils';
+import pickersMonthClasses, { getPickersMonthUtilityClass } from './pickersMonthClasses';
 
 export interface MonthProps {
+  className?: string;
   children: React.ReactNode;
+  classes?: typeof pickersMonthClasses;
```

Comment:
Since that is an internal component make sure that this prop is actually used. If not then let's remove them.

Reply:
if removed, there will be type error because `useUtilityClasses` needs `classes` from props

Reply:
> if removed, there will be type error because useUtilityClasses needs classes from props

It only needs `classes` if we want to expose them though, no? intuitively [`13a14da` (#26021)](https://github.com/mui-org/material-ui/pull/26021/commits/13a14da1d32e1602659e7b024ead858554dee013) should work just fine and doesn't expose `classes`